src/views/docs-view/server.ts

```diff
@@ -269,8 +246,7 @@ export function getStaticGenerationFunctions<
 					levelUpButtonText: `${product.name} Home`,
 				},
 				menuItems: navDataWithFullPaths as EnrichedNavItem[],
-				// TODO: won't default after `BASE_PATHS_TO_NAMES` is replaced
-				title: BASE_PATHS_TO_NAMES[basePath] || product.name,
+				title: currentRootDocsPath.shortName || currentRootDocsPath.name,
```

Comment:
With this change, do you want to go ahead and add the `"shortName": "Documentation"` property to the `docs` `rootDocsPaths` in `src/data/terraform.json`?

Reply:
Otherwise, this URL says "General Documentation" when all other /docs pages show "Documentatation": https://dev-portal-git-zsfix-terraform-nits-hashicorp.vercel.app/terraform/docs

Reply:
Oooh yeah great call 💖 ✅ updated!